src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/commands.py

```diff
@@ -5,7 +5,9 @@
 
 from azure.cli.core.commands import cli_command
 
-from ._client_factory import batch_client_factory
+from ._client_factory import batch_client_factory, batch_data_service_factory
+from ._command_type import cli_data_plane_command
+from ._validators import validate_pool_settings
```

Comment:
It is recommended to use full path instead of relative path. We do have a lot of relative path import in existing code base. I think they're bad practice, too.